pandas/core/indexes/datetimelike.py

```diff
@@ -536,7 +537,7 @@ def shift(self: _T, periods: int = 1, freq=None) -> _T:
     # --------------------------------------------------------------------
     # List-like Methods
 
-    def _get_delete_freq(self, loc: int):
+    def _get_delete_freq(self, loc: Union[int, Sequence[int]]):
```

Comment:
could be a slice too (do we have a typing alias for Indexers?)

Reply:
> (do we have a typing alias for Indexers?)

we do not.  i looked at implementing one a few weeks ago, dont remember off the top of my head why i stuck a pin in it